src/openrct2/drawing/font.c

```diff
@@ -95,6 +95,25 @@ sint32 font_sprite_get_codepoint_offset(sint32 codepoint)
     case FORMAT_SMALLDOWN: return 189 - 32;
     case FORMAT_LEFT: return 190 - 32;
     case FORMAT_INVERTEDQUESTION: return 191 - 32;
+
+    case UNICODE_A_OGONEK_UC: return RCT2_A_OGONEK_UC - 32;
```

Comment:
Can't you use character literals for these?

Reply:
Uhm, what are those again?

Also, it's been done like this throughout the codebase, not sure if it's really beneficial to change.

Reply:
Chars between apostrophes are called character literals. Instead of UNICODE_A_OGONEK_UC, you could probably write 'Ą' or perhaps u8'Ą'.

Reply:
Tried both. The first trips up Xcode, the second is not valid, apparently.